packages/orchestrator-ui-components/src/pages/processes/WFOProcessDetailPage.tsx

```diff
@@ -0,0 +1,256 @@
+import React from 'react';
+import {
+    EuiButton,
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiPageHeader,
+    EuiSpacer,
+    EuiText,
+    EuiPanel,
+} from '@elastic/eui';
+import { useTranslations } from 'next-intl';
+import { ProcessDetailStep, ProcessStatus } from '../../types';
+import {
+    parseDateRelativeToToday,
+    getProductNamesFromProcess,
+} from '../../utils';
+import { useOrchestratorTheme, useQueryWithGraphql } from '../../hooks';
+import { GET_PROCESS_DETAIL_GRAPHQL_QUERY } from '../../graphqlQueries';
+import { WFOLoading } from '../../components';
+import {
+    WFOProcessListSubscriptionsCell,
+    RenderDirection,
+} from './WFOProcessListSubscriptionsCell';
+
+interface WFOProcessDetailPageProps {
+    processId: string;
+}
+
+interface ProcessHeaderValueProps {
+    translationKey: string;
+    value: string | ProcessStatus | undefined;
+}
+
+const ProcessHeaderValue = ({
+    translationKey,
+    value = '',
+}: ProcessHeaderValueProps) => {
+    const t = useTranslations('processes.detail');
+    const { theme } = useOrchestratorTheme();
+    return (
+        <EuiFlexGroup
+            direction="column"
+            gutterSize="xs"
+            css={{
+                flex: 1,
+                whiteSpace: 'nowrap',
+                overflow: 'hidden',
+            }}
+        >
+            <EuiText size="xs">{t(translationKey)}</EuiText>
+            <EuiText
+                css={{
+                    fontWeight: theme.font.weight.bold,
+                    fontSize: theme.size.m,
+                    overflow: 'hidden',
+                    textOverflow: 'ellipsis',
+                }}
+            >
+                {value}
+            </EuiText>
+        </EuiFlexGroup>
+    );
+};
+
+export const WFOProcessDetailPage = ({
```

Comment:
The component is rather big, maybe consider to split up in smaller sub components